v2/bolt_store.go

```diff
@@ -266,3 +267,44 @@ func (b *BoltStore) GetUint64(key []byte) (uint64, error) {
 func (b *BoltStore) Sync() error {
 	return b.conn.Sync()
 }
+
+// Transition takes a BoltDB and a path, and outputs a
+// Bbolt and generates it at that path
+func Transition(old *v1.BoltStore, path string) (*BoltStore, error) {
+
+	//Create the bbolt log store
+	newbolt, err := New(Options{Path: path})
```

Comment:
Should we check if `path` already exists?  It might make sense to error out rather than risk modifying an existing database.